inc/ModelBasedPrognoser.h

```diff
@@ -55,6 +55,10 @@ namespace PCOE {
          * Produce a new prediction based on the provided data.
          */
         Prediction step(std::map<MessageId, Datum<double>> data);
+
+        const PrognosticsModel& getModel() {
+            return *model.get();
```

Comment:
This works, but C++ allows for treating `unique_ptr` like a raw pointer in cases like this. I think `return *model;` should work and would be more idiomatic.